MAUI/DataGrid/column-drag-and-drop.md

```diff
@@ -0,0 +1,226 @@
+---
+layout: post
+title: Column Drag and Drop in .NET MAUI DataGrid control | Syncfusion
+description: Learn all about column drag and drop support in Syncfusion MAUI DataGrid (SfDataGrid) control and more here.
+platform: MAUI
+control: SfDataGrid
+documentation: ug
+---
+
+# Column Drag and Drop in MAUI DataGrid (SfDataGrid)
+
+The SfDataGrid allows dragging and dropping a column header by setting the `SfDataGrid.AllowDraggingColumn` property to `true.` The drag view is displayed while dragging a column header. Based on the requirements, drag and drop operations can be handled by using the `SfDataGrid.QueryColumnDragging` event.
+
+{% tabs %}
+{% highlight xaml %}
+    <syncfusion:SfDataGrid x:Name="dataGrid"
+                           ItemsSource="{Binding OrderInfoCollection}"
+                           AllowDraggingColumn="True">
+    </syncfusion:SfDataGrid>
+{% endhighlight %}
+{% highlight c# %}
+ dataGrid.AllowDraggingColumn = true;
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="DataGrid column drag and drop" src="Images\column-drag-and-drop\maui-datagrid-column-drag-and-drop.gif" width="404"/>
+
+## Column drag and drop events
+
+The `QueryColumnDragging` event is fired continuously while dragging a column and ends when the dragging ends. By handling the `SfDataGrid.QueryColumnDragging` event, the dragging of a particular column header can be canceled.
+
+The `QueryColumnDragging` event provides the following properties in the `DataGridQueryColumnDraggingEventArgs`:
+
+- `From`: Returns the index of the currently dragging column.
+- `To`: Returns the index where you are trying to drop the column.
+- `DraggingAction`: Returns the column dragging details as a `DataGridDragAction`.
+- `DraggingPosition`: Returns the position of the drag view during column drag and drop operations.
+- `CanAutoScroll`: Returns whether auto-scrolling should happen when the column drag view reaches the left or right ends of the `SfDataGrid`.
+- `Cancel`: Returns a boolean property to cancel the event.
+
+## Cancel dragging of a particular column
+
+Dragging of a particular column can be canceled using `DataGridDragAction` argument of the `QueryColumnDragging` event handler.
```

Comment:
The dragging of a particular column can be canceled using the `DataGridDragAction` argument in the `QueryColumnDragging` event.

Reply:
Modified